vue-frontend/src/components/jobs-thankyou/ContributionSection.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="tw-my-20 lg:tw-mb-40">
```

Comment:
Review the `our success stories` section on the services page on larger(4k) devices, it should like that only